model-gen/build/calm/2024-12/core.d.ts

```diff
@@ -0,0 +1,162 @@
+export type Metadata = {
```

Comment:
These file souldn't be checked in (but doing so, for folks to review and see output)